### PR TITLE
Let cockpit subpackage depend on exact version of the main package

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec [hunk 1]
```diff
@@ -60,7 +60,8 @@ management functionality with remote management functionality.
 
 %package cockpit
 BuildArch:  noarch
-Requires:   cockpit
+Requires: cockpit
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
```
Comment (ekohl) [inline packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec:64]: Isn't `%name` sufficient?

Comment (evgeni) [reply packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec:64]: honestly? I just copied what's in the `doc` package ;)

Comment (evgeni) [reply packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec:64]: and the template:

https://github.com/theforeman/foreman-packaging/blob/8363a28d884d5ff4361bb9bc3e1eaf6d8729cfc4/gem2rpm/scl.spec.erb#L73

**Review by ekohl — APPROVED** [review]

**Review by pcreech — APPROVED** [review]
ACK

**Pull request merged**